### ISSUE TITLE
v 0.6 stable rust

### ISSUE THREAD

**Issue by awesomike** [datafuselabs/openraft#469]
Hi, is it possible to use version 0.6 with stable rust?
I get this error when I try.
```
error[E0658]: use of unstable library feature 'backtrace'
  --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/openraft-0.6.6/src/error.rs:41:17
   |
41 | #[derive(Debug, thiserror::Error)]
   |                 ^^^^^^^^^^^^^^^^
   |
   = note: see issue #53487 <https://github.com/rust-lang/rust/issues/53487> for more information
   = note: this error originates in the derive macro `thiserror::Error` (in Nightly builds, run with -Z macro-backtrace for more info)
   ```
   Is there a way to opt out of the backtrace feature?
   
   Thanks 😊 in advance

**Comment by drmingdrmer** [comment]
Sure!

But I recommend using the latest `main` branch.
`main` branch fixed several issues such as election conflict and there is a complete example for users to follow. And the API in `main` branch is more clear and simple(at least to me:))

If it's possible for you to use the `main` branch, just load openraft via git commit in your Cargo.toml:
```
[dependencies]
openraft = { git = "https://github.com/datafuselabs/openraft", rev = "db047b978e08fdb6f23bc4c27f94d4bd36ab925b" }
```

db047b978e08fdb6f23bc4c27f94d4bd36ab925b is the commit hash of the latest `main` branch.

And let me have a look at how to enable stable rust for `0.6`

**Comment by awesomike** [comment]
Thank you, very much 🙏

**Comment by drmingdrmer** [comment]
I've published v0.6.7 that will compile by stable toolchain.

But, don't you want to use the `main` branch that will be more stable than 0.6

**Comment by awesomike** [comment]
Thank, I'll stay on `main` in the mean time.
